.clang-format

```diff
@@ -1,30 +1,150 @@
 ---
-BasedOnStyle: Google
-IndentWidth: 4
-ColumnLimit: 120
-UseTab: Never
-#BreakBeforeBraces: Allman
----
-Language: Cpp
-PointerAlignment: Right
-DerivePointerAlignment: false
-# Function arguments should either be on one line, or should be each on their own line
-BinPackParameters: false
-BinPackArguments: false
-# Indent nested preprocessor directives
-IndentPPDirectives: AfterHash
-# Short conditionals are allowed:
```

Comment:
Does the new style allow or disallow these?

Reply:
These are all preserved as-is; the directives are sorted.